Dfe.Academies.External.Web/Pages/Base/BasePageEditModel.cs

```diff
@@ -41,9 +41,13 @@ public abstract class BasePageEditModel : BasePageModel
 			SchoolCacheValuesViewModel cachedValuesViewModel = new(urn, schoolDetails.EstablishmentName);
 
 			ViewDataHelper.StoreSerialisedValue(nameof(SchoolCacheValuesViewModel), ViewData, cachedValuesViewModel);
-		}
 
-		return new SchoolApplyingToConvert(schoolDetails.EstablishmentName, urn, applicationId, schoolDetails.UPRN);
+			return new SchoolApplyingToConvert(schoolDetails.EstablishmentName, urn, applicationId, schoolDetails.UPRN);
+		}
+		else
```

Comment:
Could change this if/else to a guard clause by doing the return null first:

https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html